src/sentry/api/endpoints/organization_artifactbundle_assemble.py

```diff
@@ -159,11 +156,6 @@ def post(self, request: Request, organization) -> Response:
         )
 
         if is_org_auth_token_auth(request.auth):
```

Comment:
Since in the function you already have the same if, maybe you forgot to remove it here. If, on the other hand, your goal is to avoid the function call via an upstream check (which is perfectly valid and quite idiomatic), I would suggest to remove the `if` in the function.

Reply:
hmm true, probably good enough to only check upstream! will remove the check from the function 👍